<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>143</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breadboard 8-Bit CPU Emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1" rowspan="7">
     <widget class="WidgetBus" name="_widgetBus" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="WidgetFlags" name="_widgetFlags" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="WidgetPc" name="_widgetPc" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="WidgetClock" name="_widgetClock" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="WidgetMar" name="_widgetMar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="WidgetReg" name="_widgetRam" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="WidgetIr" name="_widgetIr" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="WidgetReg" name="_widgetRegA" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="WidgetReg" name="_widgetAlu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="WidgetReg" name="_widgetRegB" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="WidgetOut" name="_widgetOut" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="WidgetSeq" name="_widgetSeq" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>143</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionMemory"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResetSoft"/>
   <addaction name="actionResetHard"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a file to run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RA&amp;M</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionResetSoft">
   <property name="text">
    <string>Soft Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset all but RAM</string>
   </property>
  </action>
  <action name="actionResetHard">
   <property name="text">
    <string>Hard Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset all</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetClock</class>
   <extends>QWidget</extends>
   <header>widgetclock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetIr</class>
   <extends>QWidget</extends>
   <header>widgetir.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMar</class>
   <extends>QWidget</extends>
   <header>widgetmar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPc</class>
   <extends>QWidget</extends>
   <header>widgetpc.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetBus</class>
   <extends>QWidget</extends>
   <header>widgetbus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetReg</class>
   <extends>QWidget</extends>
   <header>widgetreg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFlags</class>
   <extends>QWidget</extends>
   <header>widgetflags.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetOut</class>
   <extends>QWidget</extends>
   <header>widgetout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSeq</class>
   <extends>QWidget</extends>
   <header>widgetseq.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
